### ISSUE TITLE
TypeError : not all arguments converted during string formatting

### ISSUE THREAD

**Issue by r2594** [bmaltais/kohya_ss#896]
hey there,

i've just launch the new update and there is this error when I try to train a model :

```
Traceback (most recent call last):
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\gradio\routes.py", line 422, in run_predict
    output = await app.get_blocks().process_api(
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\gradio\blocks.py", line 1323, in process_api
    result = await self.call_function(
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\gradio\blocks.py", line 1051, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "C:\Users\ronan\Documents\kohya_ss\dreambooth_gui.py", line 459, in train_model
    log.info('\033[33mFolder', folder, ':', steps, 'steps\033[0m')
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 1477, in info
    self._log(INFO, msg, args, **kwargs)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 1624, in _log
    self.handle(record)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 1634, in handle
    self.callHandlers(record)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 1696, in callHandlers
    hdlr.handle(record)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 968, in handle
    self.emit(record)
  File "C:\Users\ronan\Documents\kohya_ss\venv\lib\site-packages\rich\logging.py", line 128, in emit
    message = self.format(record)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 943, in format
    return fmt.format(record)
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 678, in format
    record.message = record.getMessage()
  File "C:\Users\ronan\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py", line 368, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
```

If you guys knows what's happening, I could use some help ! Thank you very much !

Good day

**Comment by bmaltais** [comment]
It is probably something introduced with the latest kehya code update... You can go back to a previous release to get to a functional state.

**Comment by r2594** [comment]
I tried to go back to an older version but find the same problem. I might not know what version to pick. Would you be able to give me one older version ? I probably didn't pulled it correctly. I used `git clone` put I should use an other command ?

Thank you for your fast answer ! really appreciate.

**Comment by bmaltais** [comment]
Make sure to re-run setup.bat after downgrading

**Comment by r2594** [comment]
Ok, it seems to work until I start `gui-user.bat`. The `setup.bat` forces the install of torch 2.0.0 and it creates an error. as it should be 2.0.1. It crashes.

**Comment by bmaltais** [comment]
This should be fixed by v21.6.5
